packages/shared/src/hooks/usePersonalizedDigest.ts

```diff
@@ -109,6 +156,7 @@ export const usePersonalizedDigest = (): UsePersonalizedDigest => {
 
   return {
     personalizedDigest,
+    readingReminder,
```

Comment:
I am not sure it is a good idea to split the properties in return. Basically if we ever add additional type we have to then again add property and filter for that type. I would rather the consumer of the hook map and take the digest type they need. We can even expose a function for that from the hook.

Reply:
Happy to make it a func you can call by passing type?
This felt easier from consumer point, but get your remarks